### PR TITLE
show main window maximized; restore as 2/3 of primary screen

### PR DESCRIPTION
**Pull request by TarasSydor**
Given PR makes sure that GUI takes the full-screen size after startup. 
After switching to 'Normal' mode GUI takes 2/3 part of the screen, horizontally aligned.

Calculations are based on more recent QScreen instead of QDesktopWidget. It's also recommended by Qt:
https://doc.qt.io/qt-5/qdesktopwidget-obsolete.html

### CODE REVIEW

**Review by KochynVolodymyr — APPROVED** [review]
good to me

File: src/MainWindow/main_window.cpp [hunk 1]
```diff
@@ -57,12 +57,20 @@ extern const char* foedag_build_type;
...
   /* Window settings */
   setWindowTitle(tr("FOEDAG"));
-  resize(350, 250);
   m_compiler = session->GetCompiler();
   m_interpreter = session->TclInterp();
-  QDesktopWidget dw;
-  setGeometry(dw.width() / 6, dw.height() / 6, dw.width() * 2 / 3,
-              dw.height() * 2 / 3);
+
+  auto sceenGeometry = qApp->primaryScreen()->availableGeometry();
```
Comment (ravic-rs) [inline src/MainWindow/main_window.cpp:63]:
```suggestion
  auto screenGeometry= qApp->primaryScreen()->availableGeometry();
```

File: src/MainWindow/main_window.cpp [hunk 2]
```diff
@@ -57,12 +57,20 @@ extern const char* foedag_build_type;
...
+
+  // Take 2/3 part of the sceen.
+  auto mainWindowSize =
+      QSize(sceenGeometry.width() * 2 / 3, sceenGeometry.height() * 2 / 3);
+  // Center main window on the screen. It will get this geometry after switching
+  // from maximized mode.
+  setGeometry(QStyle::alignedRect(Qt::LeftToRight, Qt::AlignCenter,
+                                  mainWindowSize, sceenGeometry));
```
Comment (ravic-rs) [inline src/MainWindow/main_window.cpp:71]:
```suggestion
                                  mainWindowSize, screenGeometry));
```

File: src/MainWindow/main_window.cpp [hunk 3]
```diff
@@ -57,12 +57,20 @@ extern const char* foedag_build_type;
...
   m_interpreter = session->TclInterp();
-  QDesktopWidget dw;
-  setGeometry(dw.width() / 6, dw.height() / 6, dw.width() * 2 / 3,
-              dw.height() * 2 / 3);
+
+  auto sceenGeometry = qApp->primaryScreen()->availableGeometry();
+
+  // Take 2/3 part of the sceen.
+  auto mainWindowSize =
+      QSize(sceenGeometry.width() * 2 / 3, sceenGeometry.height() * 2 / 3);
```
Comment (ravic-rs) [inline src/MainWindow/main_window.cpp:67]:
```suggestion
      QSize(sceenGeometry.width() * 2 / 3, screenGeometry.height() * 2 / 3);
```

**Review by ravic-rs — CHANGES_REQUESTED** [review]

**Review by ravic-rs — APPROVED** [review]
Looks good.

**Pull request merged**